lms/djangoapps/courseware/toggles.py

```diff
@@ -38,3 +38,32 @@
 COURSEWARE_MICROFRONTEND_COURSE_TEAM_PREVIEW = CourseWaffleFlag(
     WAFFLE_FLAG_NAMESPACE, 'microfrontend_course_team_preview', __name__
 )
+
+COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES = ExperimentWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'mfe_progress_milestones', __name__)
```

Comment:
Are we experimenting this feature? I think the Experiment Waffle Flag is more for doing like 50/50 rollouts and not just on/off switches. We might just want a CourseWaffleFlag here.